rc/tools/man.kak

```diff
@@ -130,18 +125,51 @@ define-command man-link %{ evaluate-commands -save-regs / %{
 
 define-command -docstring 'Try to jump to a man page' \
 man-jump %{
-  try %{ man-link } catch %{ man-link-here } catch %{ fail 'Not a valid man page link' }
-  try %{ man } catch %{ fail 'No man page link to follow' }
+  try %{
```

Comment:
I think we could just expand the man_link1 option to contain that line-break alternative, and tweak the `man` command implementation to strip them from the selection, should just be sed oneliner.

Reply:
Any issue with that alternaive approach ? I think it would avoid all this quite complex logic.